ghost/core/core/server/data/schema/schema.js

```diff
@@ -1021,5 +1021,22 @@ module.exports = {
         currency: {type: 'string', maxlength: 24, nullable: true},
         created_at: {type: 'dateTime', nullable: false},
         email_sent_at: {type: 'dateTime', nullable: true}
+    },
+    collections: {
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        title: {type: 'string', maxlength: 191, nullable: false},
+        slug: {type: 'string', maxlength: 191, nullable: false, unique: true},
+        description: {type: 'string', maxlength: 2000, nullable: true},
+        type: {type: 'string', maxlength: 50, nullable: false},
```

Comment:
We'd like to have this validated straight away imo. The two allowed variables right now would be `automatic` and `manual`

Reply:
Validation is in the Entity rather than the bookshelf model - I don't think there's much point duplicating that?